spec/caching_spec.rb

```diff
@@ -29,16 +29,16 @@
     end
 
     it 'caches per within the current thread' do
-      expect(subject.fire(http_stats_url('200'))).to_not be_cached
-      expect(subject.fire(http_stats_url('200'))).to be_cached
-      expect(subject.fire(http_stats_url('200'))).to be_cached
+      expect(subject.fire('www.example.com')).to_not be_cached
```

Comment:
httpstat.us has been timing out lately. And apparently this spec needs to make lie calls (VCR messes up the caching). Replaced it with example.com for now, but the best option would probably be a local server.